client/client.go

```diff
@@ -268,9 +268,9 @@ func (c *Client) Containers(ctx context.Context, filters ...string) ([]Container
 	if err != nil {
 		return nil, err
 	}
-	var out []Container
```

Comment:
There is option with less changes.

out := make([]Container, 0, len(r))

Reply:
Ah, yeah, I usually take that approach myself (0 initial size, but set capacity to the expected size); at least it's slightly less error prone in cases where the loop _may_ contain filtering (`continue` / otherwise).

In all honesty, I have not benchmarked if that is less performant though; perhaps I should at some point 😂

Reply:
Agreed. We use the form I went with in this change elsewhere in the project quite often so that's why I chose this, but I didn't check for if we use the (type, len, cap) constructor + append much to be honest.